distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -137,6 +137,16 @@ org.codice.allowBasicAuthOverHttp=false
 
 # Sets the default port number for the catalog-ftp feature FTP endpoint
 org.codice.ddf.catalog.ftp.port=8021
+#
+# Maximum Endpoint Contact Interval
+#
+# Max seconds between attempts to contact the endpoint of a source
+org.codice.ddf.platform.util.http.maxRetryInterval=300
```

Comment:
My mistake. I read the increment code too quickly and thought it was taking _min_, not _max_. This is fine.